### PR TITLE
Update classifier-learn-about.md

### PR DESCRIPTION
**Pull request by edwinchan-microsoft**
added language support details.

### CODE REVIEW

**Comment by edwinchan-microsoft** [comment]
@chrfox

File: microsoft-365/compliance/classifier-learn-about.md [hunk 1]
```diff
@@ -131,6 +131,8 @@ Pre-trained classifiers can scan content in these languages:
 - Portuguese
 - Spanish
 
+All others are English only at the moment.
+
 ### Custom classifiers
 
 When the pre-trained classifiers don't meet your needs, you can create and train your own classifiers. There's significantly more work involved with creating your own, but they'll be much better tailored to your organizations needs.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/classifier-learn-about.md]:
```suggestion
When the pre-trained classifiers don't meet your needs, you can create and train your own classifiers. There's significantly more work involved with creating your own, but they'll be much better tailored to your organization's needs.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @edwinchan-microsoft, not related, but kindly also commit the additional suggested change to correct the minor typo. Many thanks.

**Comment by yogkumgit** [comment]
@edwinchan-microsoft Gentle remindedr

**Pull request merged**